src/js/actions/SourceContentUpdatesActions.js

```diff
@@ -103,6 +105,7 @@ export const downloadSourceContentUpdates = (languageIdListToDownload) => (async
     await SourceContentUpdater.downloadResources(languageIdListToDownload, USER_RESOURCES_PATH)
       .then(async () => {
         updateSourceContentUpdaterManifest();
+        dispatch(updateSourceContentUpdatesReducer());
```

Comment:
Why not add dispatch(updateSourceContentUpdatesReducer()) inside updateSourceContentUpdaterManifest() so that you only have to add it once? would that make sense?